### ISSUE TITLE
Infinite reels from crafting table

### ISSUE THREAD

**Issue by otavio-silva** [mrh0/createaddition#401]
**Describe the bug**
I can create infinite  reels from nothing in the crafting table, it's like the recipe is empty

**To Reproduce**
- Create crafting table
- Click on the 3x3 grid on the crafting table with nothing in it
- Can create 16 reels from nothing

**Additional context**
- Mod version: [20221128a](https://www.curseforge.com/minecraft/mc-mods/createaddition/files/4120847)
- Fabric version: 0.14.10
- Create version: [v0.5.0g Patch 1](https://www.curseforge.com/minecraft/mc-mods/create-fabric/files/4125637)

**Comment by mrh0** [comment]
The recipe looks to be right, dont know whats causing this
```
{
	"type": "minecraft:crafting_shaped",
	
	"pattern":
	["P",
	 "N",
	 "P"],
	"key":
	{
		"P":
		{
			"tag": "c:plates/iron"
		},
		"N":
		{
			"tag": "c:nuggets/iron"
		}
	},
	"result":{
		"item": "createaddition:spool",
		"count": 16
	}
}
```

**Comment by otavio-silva** [comment]
I found out it was the FastBench mod (https://www.curseforge.com/minecraft/mc-mods/fastbench-for-fabric) that I was using and it's not properly updated to 1.19.2, removing it and the error is gone.

**Comment by mrh0** [comment]
Fixes in 20221219a.
